drafts/latest/tables/recommended-properties-for-metric.html

```diff
@@ -0,0 +1,62 @@
+<table class="simple" id="table-recommended-properties-for-metric">
+<thead>
+<tr>
+<th>
+<p>Property</p>
+</th>
+<th>
+<p>URI</p>
+</th>
+<th>
+<p>Range</p>
+</th>
+<th>
+<p>Usage note</p>
+</th>
+<th>
+<p>Card.</p>
+</th>
+</tr>
+</thead>
+<tbody>
+
+<tr class="property recommended ap-ext" id="metric-dimension">
+<td>
+<p>+dimension</p>
+</td>
+<td>
+<p><code>dqv:inDimension</code></p>
+</td>
+<td>
+<p><code>dqv:Dimension</code></p>
+</td>
+<td>
+<p>This property represents the dimensions a quality metric, certificate and annotation allow a measurement of.</p>
+<p>In GeoDCAT-AP, this property is used in the definition of the instances of <code>dqv:Metric</code> corresponding to the different type of spatial resolution. It MUST always take as value <code>dqv:precision</code>.</p>
```

Comment:
What if in the future or in some profile of GeoDCAT-AP, DQV is used also for other quality dimensions? I would consider contextualizing more the 'MUST', for example, rephasing as in the following. 
```suggestion
<p>In GeoDCAT-AP, this property is used in the definition of the instances of <code>dqv:Metric</code> corresponding to the different type of spatial resolution. When referring to the different type of spatial resolution, it MUST always take as value <code>dqv:precision</code>.</p>
```

Reply:
Good point, @riccardoAlbertoni . That was actually the intended meaning, but the wording was bad.